### PR TITLE
Remove extra whitespaces in help text of some subcommands

### PR DESCRIPTION
**Pull request by tkhoa2711**
**What I did**

A relatively small and simple change to just remove extra leading whitespaces in help text of the `run` and `down` subcommands.

What it currently looks like before the change (note the second last line):

```
> docker compose down --help

Usage:  docker compose down

Stop and remove containers, networks

Options:
      --remove-orphans    Remove containers for services not defined in the Compose file.
      --rmi string        Remove images used by services. "local" remove only images that don't have a custom tag ("local"|"all")
  -t, --timeout int       Specify a shutdown timeout in seconds (default 10)
  -v, --volumes volumes    Remove named volumes declared in the volumes section of the Compose file and anonymous volumes attached to
                          containers.
```

**Related issue**
I didn't create an issue since this is a relatively small.

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thank you for contributing! It appears your commit message is missing a DCO sign-off,
causing the DCO check to fail.

We require all commit messages to have a `Signed-off-by` line with your name
and e-mail (see ["Sign your work"](https://github.com/moby/moby/blob/v20.10.11/CONTRIBUTING.md#sign-your-work)
in the CONTRIBUTING.md in this repository), which looks something like:

```
Signed-off-by: YourFirsName YourLastName <yourname@example.org>
```

There is no need to open a new pull request, but to fix this (and make CI pass),
you need to _amend_ the commit(s) in this pull request, and "force push" the amended
commit.

Unfortunately, it's not possible to do so through GitHub's web UI, so this needs
to be done through the git commandline.

You can find some instructions in the output of the DCO check (which can be found
in the "checks" tab on this pull request), as well as in the [Moby contributing guide](https://github.com/moby/moby/blob/v20.10.11/docs/contributing/set-up-git.md).

Steps to do so "roughly" come down to:

1. Set your name and e-mail in git's configuration:

    ```bash
    git config --global user.name "YourFirstName YourLastName"
    git config --global user.email "yourname@example.org"
    ```

    (Make sure to use your _real_ name (**not your GitHub username/handle**) and e-mail)


2. Clone your fork locally
3. Check out the branch associated with this pull request
4. Sign-off and amend the existing commit(s)

    ```bash
    git commit --amend --no-edit --signoff
    ```

    If your pull request contains multiple commits, either squash the commits (if
    needed) or sign-off each individual commit.

5. _Force push_ your branch to GitHub (using the `--force` or [`--force-with-lease`](https://stackoverflow.com/questions/52823692/git-push-force-with-lease-vs-force) flags) to update the pull request.


Sorry for the hassle (I wish GitHub would make this a bit easier to do), and let me know if you need help or more detailed instructions!

**Review by thaJeztah — APPROVED** [review]
LGTM

The change itself looks good to me, let me start CI to verify if the docs also need to be regenerated (not sure if it strips the whitespace when generating the markdown files); you can re-generate the docs with `make docs`

**Comment by tkhoa2711** [comment]
Looks like I will need to update the markdown docs as well

**Comment by tkhoa2711** [comment]
@thaJeztah I've updated the docs as well. Another CI trigger approval would be appreciated. Thanks!

**Review by thaJeztah — APPROVED** [review]
LGTM (not a maintainer, so "non-binding" 😅)

**Review by milas — APPROVED** [review]

**Pull request merged**
